nengo_gui/static/sim_control.css

```diff
@@ -122,3 +122,24 @@
     padding: 0.2em !important;
     border: none !important;
 }
+
+#speed_throttle {
+    height: 10px;
+    width: 110px;
+    #background: yellow;
```

Comment:
Is there any reason to have the # in front of the background property? As far as I can tell its only for IE support.